Avatar/Badge/Badge.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Parcel Sandbox</title>
+    <meta charset="UTF-8" />
+    <link
+      href="https://fonts.googleapis.com/icon?family=Material+Icons"
```

Comment:
You could provide all imports (external) in main component library CSS file rather than in an individual file.

Reply:
I will do it once I merge everything to dev. Please approve, if we are good to go